canonical-websites/vanillaframework.io
# Add a status check page at /_status/check

This is to conform to the URL structure used by Talisker, so we can
have a consistent way to check if our apps are up in Kubernetes.

This will make https://github.com/canonical-webteam/deployment-configs/pull/149 work.

QA
--

`./run`

Then go to http://127.0.0.1:8014 to check the site still works, and go to http://127.0.0.1:8014/_status/check, and check you see "OK". Or check http://vanillaframework.io-pr-143.run.demo.haus/_status/check.
